styles.css

```diff
@@ -0,0 +1,628 @@
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        4.3 Projects
+        4.4 Services
+        4.5 Form
+        4.6 Footer
+
+      5. Media Queries */
+
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+/*Paars #5722CD;*/
+/*Blauw #4845E4;*/
+/*Roze #ED4385;*/
+/*Zwart #0C0C0C (voor standaard tekst);*/
+/*Lichtpaars #F2F0FF (voor de grote zwevende woorden, zoals "hello" en "services")
+Uitzonderingen:*/
+/*Lichtgrijs #F3F6F9 (voor de achtergrond van de projecten)*/
+/*!*Grijs #7A7A7A (voor de service-tegel tekst); */
+/*font-family: 'Merriweather', serif;*/
+/*font-family: 'Roboto', sans-serif;*/
+
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    /*border: 1px solid indianred;*/
+}
+
+:root {
+    --standard-text-black: #0C0C0C;
+    --bg-text-lightpurple: #F2F0FF;
+    --bg-color-blue: #4845E4;
+    --color-purple: #5722CD;
+    --button-color-pink: #ED4385;
+    --bg-projects-lightgrey: #F3F6F9;
+    --tiles-text-grey: #7A7A7A;
+
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+body {
+    font-family: 'Roboto', Arial, sans-serif;
+    text-align: center;
+    font-weight: normal;
+    color: var(--standard-text-black);
+    min-height: 100vh;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Merriweather', Arial, serif;
+    font-weight: normal;
+}
+
+html {
+    font-size: 16px;
+}
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+.outer-container {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+}
+
+.inner-container {
+    display: flex;
+    flex-direction: column;
+    flex: 0 1 1200px;
+    margin-inline: 1rem;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    gap: 1rem;
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+
+a {
+    text-decoration: none;
+}
+
+h1 {
+    font-size: 1.5rem;
+}
+
+h2 {
+    font-size: 1.3rem;
+}
+
+h3 {
+    font-size: 1rem;
+}
+
+h4 {
+    font-size: 1.2rem;
+}
+
+h5 {
+    font-size: 1.2rem;
+}
+
+h6 {
+    font-size: 1rem;
+}
+
+p {
+    font-size: 1rem;
+}
+
+button {
+    background-color: var(--button-color-pink);
+    border-radius: 50px;
+    padding: 1.2rem 1.4rem;
```

Comment:
Als 1 rem neer komt op 16px binnen je applicatie, dan komen 1.2 en 1.4rem uit op een kommagetal qua pixels. Dit is niet fijn voor browsers om te renderen. Probeer daarom uit te komen op hele pixels.